builds/ZIKA.md

```diff
@@ -13,6 +13,25 @@
   * Select year >= 2013 and genome length >= 5000
   * Download as Genome Fasta
   * Set Custom Format Fields to 0: GenBank Accession, 1: Strain Name, 2: Segment, 3: Date, 4: Host, 5: Country, 6: Subtype, 7: Virus Species
+  * May also use the [ViPR API](https://www.viprbrc.org/brc/staticContent.spg?decorator=reo&type=ViprInfo&subtype=API)
+
+  ```
+  curl -k "https://www.viprbrc.org/brc/api/sequence?datatype=genome&family=flavi&species=Zika%20virus&fromyear=2003&minlength=5000&metadata=genbank,strainname,segment,date,host,country,genotype,species&output=fasta" |\
```

Comment:
We shouldn't recommend `-k` (`--insecure`) here.

Reply:
Ah, good catch. I'll remove `-k`

Reply:
Ended up changing it as a separate commit. `master` branch is protected from git squash

```
fauna % git push -f origin
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 4 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 914 bytes | 914.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: Cannot force-push to this protected branch
To https://github.com/nextstrain/fauna.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'https://github.com/nextstrain/fauna.git'
```

Reply:
Good practice to avoid/prevent rewriting history on `master` since that can mess things up! Either push a new commit like you did and mention change somewhere (ok) or create a new PR if not in a rush to get the change in (even better).